README.markdown

```diff
@@ -111,6 +124,15 @@ $ sudo make install
 $ sudo ldconfig
 ```
 
+#### Building against non-installed libraries
+
+If you wish to build STP's dependencies without installing them, you can tell CMake where to find the non-installed artefacts; for example:
+
+* ``-DMINISAT_INCLUDE_DIRS:PATH=<path>`` and ``-DMINISAT_LIBDIR:PATH=<path>`` -- the paths to ``minisat/core/Solver.h`` and the ``minisat`` libraries (respectively)
```

Comment:
In Markdown only one backtick is needed to make text monospaced.

Reply:
Hahaha, yeah, I know ... weirdly, there were other instances of double-back tick in this markdown, so I copied it for consistency. I'll fix it everywhere.

Reply:
Done!